std/format.d

```diff
@@ -569,66 +569,70 @@ matching failure happens.
 
 Throws:
     An `Exception` if `S.length == 0` and `fmt` has format specifiers
+
+Deprecated:
+
+The deprecation targets parameters of the form `&param` which take the address of
+a local and make most usages of this function non-@safe. Replace these parameters
+simply by `param`.
+Paramters of the form `paramFunc()` are false positives; replace them by
+`*paramFunc()` to surpass the deprecation warning correctly.
```

Comment:
Hmm this could be quite difficult to justify. Have you tried to add this to the if constraints?